Update dbt-python-bigframes.md (#8131)
fishtown-analytics/docs.getdbt.com: website/docs/guides/dbt-python-bigframes.md

- Basic to intermediate SQL and python.
- Basic understanding of dbt fundamentals. We recommend the [dbt Fundamentals course](https://learn.getdbt.com).

> During setup, you’ll need to select the **BigQuery (Legacy)** adapter and enter values for your **Google Cloud Storage Bucket** and **Dataproc Region** in <Constant name="dbt_platform"/>. See [Configure BigQuery in dbt platform](/guides/dbt-python-bigframes?step=2#configure-bigquery-in-dbt-platform) for details.

Reviewer: ```suggestion
During setup, you’ll need to select the **BigQuery (Legacy)** adapter and enter values for your **Google Cloud Storage Bucket** and **Dataproc Region** in the <Constant name="dbt_platform"/>. See [Configure BigQuery in dbt platform](/guides/dbt-python-bigframes?step=2#configure-bigquery-in-dbt-platform) for details.
```

   gcloud storage buckets add-iam-policy-binding gs://${GOOGLE_CLOUD_PROJECT}-bucket --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/storage.admin
   ```

> ### Configure BigQuery in dbt platform

Reviewer: ```suggestion
### Configure BigQuery in the dbt platform
```

> When setting up your BigQuery DataFrames connection in the <Constant name="dbt_platform"/>, make sure you:

Reviewer: The previous wording sounds like a prerequisite to me, suggesting this instead:
```suggestion
To set up your BigQuery DataFrames connection in the <Constant name="dbt_platform"/>, refer to the following steps:
```

> 1. Select **BigQuery (Legacy)** as your adapter.

Reviewer: Is the user creating a new connection here or is there an existing BigQuery connection? I checked the Introduction of the guide and it seems that this is a new connection. Can we add the steps here:

```suggestion
1. Go to **Account settings** > **Connections**. Click **New connection**. 
2. In the **Type** section, select **BigQuery**.
3. Select **BigQuery (Legacy)** as your adapter.
```

So that the user has an idea where to go to set up the adapter

> 2. Under **Optional Settings**, enter values for the following fields:

Reviewer: ```suggestion
2. Under **Optional settings**, enter values for the following fields:
```

    - **Google Cloud Storage Bucket** (for example: `dbt_name_bucket`)
    - **Dataproc Region** (for example: `us-central1`)
3. Click **Save**.
 
This is required so that BigFrames jobs execute correctly.

> Refer [Connect to BigQuery](/docs/cloud/connect-data-platform/connect-bigquery) for more info on how to connect to BigQuery in <Constant name="dbt_platform"/>.

Reviewer: ```suggestion
Refer to [Connect to BigQuery](/docs/cloud/connect-data-platform/connect-bigquery) for more info on how to connect to BigQuery in the <Constant name="dbt_platform"/>.
```

> <Lightbox src="/img/guides/gcp-guides/dbt-platform-bq.png" width="30%" title="Configure BigQuery in dbt platform"/>

Reviewer: ```suggestion
<Lightbox src="/img/guides/gcp-guides/dbt-platform-bq.png" width="30%" title="Configure BigQuery in the dbt platform"/>
```

   gcloud storage buckets add-iam-policy-binding gs://${GOOGLE_CLOUD_PROJECT}-bucket --member=serviceAccount:dbt-bigframes-sa@${GOOGLE_CLOUD_PROJECT}.iam.gserviceaccount.com --role=roles/storage.admin
   ```

### Configure BigQuery in the dbt platform

To set up your BigQuery DataFrames connection in the <Constant name="dbt_platform"/>, refer to the following steps:
1. Go to **Account settings** > **Connections**. Click **New connection**. 
2. In the **Type** section, select **BigQuery**.
3. Select **BigQuery (Legacy)** as your adapter.
2. Under **Optional settings**, enter values for the following fields:
    - **Google Cloud Storage Bucket** (for example: `dbt_name_bucket`)
    - **Dataproc Region** (for example: `us-central1`)
3. Click **Save**.
 
This is required so that BigFrames jobs execute correctly.

Refer to [Connect to BigQuery](/docs/cloud/connect-data-platform/connect-bigquery) for more info on how to connect to BigQuery in the <Constant name="dbt_platform"/>.

> <Lightbox src="/img/guides/gcp-guides/dbt-platform-bq.png" width="30%" title="Configure BigQuery in the dbt platform"/>

Reviewer: Adjusting image size:
```suggestion
<Lightbox src="/img/guides/gcp-guides/dbt-platform-bq.png" width="80%" title="Configure BigQuery in the dbt platform"/>
```
